app/Http/routes.php

```diff
@@ -249,6 +249,7 @@
     Route::get('event/{id}/gallery/upload','Event\EventGalleryController@add');
     Route::post('event/{id}/gallery/upload','Event\EventGalleryController@upload');
     Route::post('event/{id}/gallery','Event\EventGalleryController@store');
+    Route::post('event/deletephoto/{id}','Event\EventGalleryController@destroy');
```

Comment:
`event/{id}/gallery/{photo_id}/destroy`. The method will be `delete` instead of `post`